go.mod

```diff
@@ -9,11 +9,19 @@ require (
 	github.com/infobloxopen/seal v0.2.3
 	github.com/open-policy-agent/opa v0.37.2
 	github.com/sirupsen/logrus v1.8.1
+	github.com/stretchr/testify v1.7.0
 	go.opencensus.io v0.23.0
 	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
 	google.golang.org/grpc v1.44.0
 )
 
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.1.1 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
are these new packages required for http middleware ?

Reply:
[out.txt](https://github.com/infobloxopen/atlas-authz-middleware/files/14163452/out.txt)
this is the graph file for the dependencies, as per the file, these might have been introduced due to `go.uber.org/multierr` or `github.com/stretchr/testify`. Middleware is using these libraries

Reply:
this is middleware, lets use standard go libraries.

Reply:
Hi @drewwells , I see that these two libraries are widely used in infoblox.
https://github.com/search?q=org%3AInfoblox-CTO%20github.com%2Fstretchr%2Ftestify&type=code

https://github.com/search?q=org%3AInfoblox-CTO%20go.uber.org%2Fmultierr&type=code